clients/iscmove/isc.go

```diff
@@ -77,7 +77,7 @@ type AssetsBagBalances map[iotajsonrpc.CoinType]iotajsonrpc.CoinValue
 
 type AssetsBagWithBalances struct {
 	AssetsBag
-	Balances AssetsBagBalances `bcs:"-"`
+	Balances AssetsBagBalances `bcs:"optional"`
```

Comment:
Why is it optional? Maybe explain in a comment?

Reply:
Only to save space. With that you are at least free to decide if you want it or not.

Reply:
Seems error prone. Why not make it mandatory, and if you want to sae space just encode the AssetsBag?

Reply:
fine